content/terms_and_definitions/terms_and_definitions.adoc

```diff
@@ -6,7 +6,86 @@ ifndef::include-only-once[]
+
+3D Environment::
+A 3D representation of the surrounding of a simulated sensor.
+The 3D environment can consist of multiple 3D assets.
+
+Co-Simulation::
+Simulating multiple independent models connected with interfaces.
+A co-simulation platform handles the timing and data transfer between the individual models
+
+Ground truth::
+True static and dynamic states of all objects as defined by content creator or observed by omniscient observer (e.g. simulation platform).
+
+Material::
+A collection of material properties belonging to one entity.
+
+Material assignment::
+A mapping of material IDs to parts of 3D models.
+The material IDs are unique a point to a set of material properties in separate material files.
+
+Material classification::
+Standardized classification scheme that sorts materials into classes and groups of classes, for example, based on their physical properties.
+
+Material Parameter File::
+A file in json format containing defined material properties according to the OpenMATERIAL standard.
+
+Material property::
+Intensive physical or chemical properties of a material 
+
+* Notes: Open Point: What about geometric properties, i.e. roughness, thickness?
+
+Multispectral/hyperspectral::
+A wide range of wavelengths.
+
+Normals (face, vertex normals)::
+A vector that is perpendicular to a given object, surface, vertex or geometrical primitive.
+
+Object::
+An entity to which an object ID can be assigned.
+It comprises discrete meshes and defined 3D boundaries.
+
+Object Part / Subtype::
+(To be defined)
+
+* Notes: What is an object and what is an object part?
+
+Object Type::
+Class of objects with common properties
+
+Perception::
+The process of detecting objects and generating situational awareness of the surrounding of an automated system using perception sensors.
+"Machine perception is to reliably detect all the other road users relevant to the operation of the automated driving, and to assign them correctly to the traffic infrastructure."
+
+* Source: Klaus Dietmayer, Predicting of Machine Perception for Automated Driving, 2016
+
+Physical (as in physical parameter)::
+A physical property is any property of a physical system that is measurable. 
+
+* Source: Mark, Burgin (2016-10-27). Theory Of Knowledge: Structures And Processes
+
+Post (Rendering) Effects::
+Rendering techniques to improve fidelity while maintaining performance, that are typically non-physical (or in other words fake)
+
+Scenario::
+A description of the behavior or temporal evolution of physical objects and environmental conditions on the driving infrastructure over an interval of time, including the movement of traffic participants or the change of environmental conditions.
+
+* Source: ASAM OpenSCENARIO v1.3.0 standard, 2024
+
+Signal (Energy) Propagation::
+"During signal [or energy] propagation, the signal travels through the environment and possibly interacts with objects and particles."
```

Comment:
Proposal: Rephrase the quote to an indirect one as to match the other definitions' template

The act of a signal traveling through the environement and potentially interacting with objects and particles.

Reply:
done